rviz_common/src/rviz_common/add_display_dialog.cpp

```diff
@@ -613,10 +613,12 @@ QTreeWidgetItem * TopicDisplayWidget::insertItem(
     bool match = false;
     for (int c = 0; c < current->childCount(); ++c) {
       QTreeWidgetItem * child = current->child(c);
-      if (child->text(0) == part && !child->data(1, Qt::UserRole).isValid()) {
-        match = true;
-        current = child;
-        break;
+      if (child != NULL) {
```

Comment:
Please either use `if (child) {` or `if (child == nullptr) {`.

Reply:
Also please combine it into the condition on the next line rather than adding another indention:

```
if (child && child->text(0) == part && !child->data(1, Qt::UserRole).isValid()) {
```

Reply:
Thanks for review, skill learned.